Documentation/conf.py

```diff
@@ -77,11 +77,16 @@
 if branch is None or branch == 'latest':
     branch = 'HEAD'
     archive_name = 'master'
+    chart_release = './cilium'
 elif branch == 'stable':
     branch = release
     archive_name = release
+    chart_release = 'cilium/cilium --version ' + release
+    tags.add('stable')
 else:
     archive_name = branch
+    chart_release = 'cilium/cilium --version ' + branch
```

Comment:
I think we had a similar discussion before, but now that I'm testing this I can see that this can point to something like `v1.7` which we will likely not generate helm templates for.

This `branch` should probably be `release` as well. Then on the release side we just need to make sure that the `VERSION` file always contains the most recently released version for stable branches.